slf4j/src/main/scala/io/chrisdavenport/log4cats/slf4j/Slf4jLoggerFactory.scala

```diff
@@ -0,0 +1,17 @@
+package io.chrisdavenport.log4cats.slf4j
+
+import cats.effect.Sync
+import io.chrisdavenport.log4cats.{LoggerFactory, SelfAwareStructuredLogger}
+
+class Slf4jLoggerFactory[F[_]: Sync] extends LoggerFactory[F] {
```

Comment:
If we could put this in the companion object and make it private then we don't need to let anything besides a LoggerFactory interface escape.